test/components/views/location/Map-test.tsx

```diff
@@ -107,10 +107,10 @@ describe("<Map />", () => {
         });
 
         it("updates map center when centerGeoUri prop changes", () => {
-            const component = getComponent({ centerGeoUri: "geo:51,42" });
+            const { rerender } = getComponent({ centerGeoUri: "geo:51,42" });
 
-            component.setProps({ centerGeoUri: "geo:53,45" });
-            component.setProps({ centerGeoUri: "geo:56,47" });
+            getComponent({ centerGeoUri: "geo:53,45" }, rerender);
+            getComponent({ centerGeoUri: "geo:56,47" }, rerender);
```

Comment:
Not sure if I'm misreading this, but could you call `rerender` here as 
```
rerender({ centerGeoUri: "geo:53,45" })
etc
```
then remove the second argument to getComponent? Perhaps I'm missing something

Reply:
`rerender` requires me to copy all the JSX and the other props, this re-uses the previous  helper.

Reply:
Ah yes, that's right. I thought I was missing something